sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -418,7 +442,14 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       .where("programs.programStatus = :programStatus", {
         programStatus: ProgramStatus.Approved,
       })
-      .andWhere("programs.isActive = true");
+      .andWhere("programs.isActive = true")
+      .andWhere(
+        new Brackets((qb) => {
+          qb.where("programs.effectiveEndDate > :currentDate", {
```

Comment:
We also used the DB `CURRENT_DATE` in other queries instead of adding a parameter.
`CURRENT_DATE` will also not include time information.